yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -141,7 +141,7 @@ export class AvmPersistableStateManager {
    * @param value - the value being written to the slot
    */
   public writeStorage(storageAddress: Fr, slot: Fr, value: Fr) {
-    this.log.debug(`storage(${storageAddress})@${slot} <- ${value}`);
+    this.log.info(`Storage write (address=${storageAddress}, slot=${slot}): value=${value}`);
```

Comment:
Is the current log in public_executor also at info level?

Reply:
I was wrong! It's debug level. Fixing this....